concordium-sdk/src/main/java/com/concordium/sdk/responses/accountinfo/credential/Attributes.java

```diff
@@ -5,6 +5,10 @@
 import lombok.Getter;
 import lombok.ToString;
 
+/**
+ * The attributes associated with an `Identity`
+ * The attributes are commitments if they are present for the `Credential`.
```

Comment:
I'm not sure I get this comment. Attributes can either be just commitments, or they are revealed.

Reply:
I agree. I don't recall what my thought was behind this latter comment. I'll just remove it for now.